### PR TITLE
Balance: reduce the damage and increase the blast radius of heavy rockets, making them less likely to one-shot small ships and giving them a clearer role as an anti-swarm weapon.

### PR DESCRIPTION
**Pull request by Quantumshark**
**Balance**

## Summary
Reduces the damage dealt by heavy rockets from 1060/900 to 790/670, and increase their blast radius from 50 to 90. This is a drop of about 25% to damage, and an increase of 80% to blast radius.

Heavy rockets are, currently, too lethal for small ships. They're the cause of a lot of sudden deaths in the early game. In theory, their low speed makes them easy to dodge, but in practice that disadvantage is rarely visible because highly-mobile interceptors deploy them at point-blank range, giving no time to dodge them. With the larger blast radius, those interceptors will have to attack from slightly further away, giving a little more time to react - and because the trigger radius is unchanged, the extra blast radius doesn't actually make evading any harder. So, now they're easier to dodge, and less likely to ouright kill you if you don't.
In exchange for that weakness, they get a larger spread of damage, making them more effective when deployed against clustered foes - the explosion now covers a little over three times the area it previously did, so the total damage inflicted in anti-fleet scenarios has the potential to be much higher despite the lower damage dealt to each ship.

The description of heavy rockets notes that they're primarily effective against groups of fighters, and this change leans further into that, making them more clearly distinct from the Torpedo, the premier anti-HW weapon for small ships.

### CODE REVIEW

**Comment by Quantumshark** [comment]
Specific ship notes:
One heavy rocket will now almost, but not quite, disable a shuttle, whereas previously it would leave one disabled on ~25hp.
Two heavy rockets will now not quite disable a sparrow, whereas previously they would have destroyed it.

**Review by Hurleveur — APPROVED** [review]
Yes! Finally!
I think the way we finish fixing this is AI related, by making them fire those weapons before they're at point blank range, but the big drop in dmg is really appreciated
Perhaps we could tweak the hull/shield dmg ratio, too?

**Comment by Zitchas** [comment]
I am of the opinion that it is silly and counter-intuitive to have the thing for dealing with groups of fighters be unguided, while the thing for dealing with heavy warships *is*. Yes, torpedoes are the premier HW vs HW weaponry. A heavy rocket is basically tailored specifically for small ships to use as a anti-HW weapon. Light ships don't need guided weaponry, as they can aim very well. Therefore they are perfectly suited for deploying dumbfire weapons that sacrifice guidance for heavier payloads. 

Likewise, as many people have pointed out, it is extremely easy to kill light ships. We don't need a specialty weapon to kill more of them faster. On the other hand, we lack weapons suited for enabling light ships to take out (or at least heavily damage) big ships.

Beyond this, I think a large part of the problem with heavy rockets killing players in the early game is that pirates carry them for no discernible reason. They should at least have some *semblance* of being optimized to disable ships, instead of being outfitted like assassins focused on blowing everything into component atoms as quickly as possible.

In short, I think the best solution to this "problem" is to reduce the frequency of heavy rockets in pirate fleets, and change the description of the heavy rocket to describe its primary use as being the tool used by interceptors and fighters to threaten and eliminate larger warships, and especially to large groups of large ships.

**Comment by Quantumshark** [comment]
> I am of the opinion that it is silly and counter-intuitive to have the thing for dealing with groups of fighters be unguided, while the thing for dealing with heavy warships is.

Guided weapons are, on the whole, dodgeable - they can be seen coming, they generally come from a long enough distance to provide reaction time. This makes them more suitable for use against slower, less mobile ships, and less effective against small fast ships. Conversely, an unguided weapon must be used at close or medium range, where there's less time to avoid it, allowing it to be effective against smaller ships in a way that seeking weapons are not.

> Yes, torpedoes are the premier HW vs HW weaponry. A heavy rocket is basically tailored specifically for small ships to use as a anti-HW weapon.

> On the other hand, we lack weapons suited for enabling light ships to take out (or at least heavily damage) big ships.

Torpedoes (in pod form) are also ideal weapons for light ships to use against heavy warships. Small, mobile ships can take advantage of the long range of torpedoes, or can strike with them at point-blank range in a high-speed flyby thanks to their lack of a blast radius. They're far more useful for a small mobile ship than heavy rockets are. They're far more useful for any ship than heavy rockets are, because they're able to be used at long range or point-blank range, rather than exclusively at medium ranges.
Heavy rockets, on the other hand, have a blast radius and trigger radius, which means they don't need to score a direct hit. That makes them ideal as a weapon for less mobile ships to use against more mobile ships, rather than the reverse.

So, there's already an ideal anti-HW weapon for smaller ships to use. We do not, therefore, need heavy rockets to be more of the same thing. Instead, heavy rockets should lean into something that nothing else does, to provide a wider range of options for the player. Being anti-fleet, crowd-control weapons makes the best use of their unique (among human weapons) blast radius, and fits their description.

**Comment by Hurleveur** [comment]
I agree w QuantumShark, also a weapon against a group of heavy warship doesnt seem like it'd be useful except against the player, and even then it'd be a joke compared to things like nukes

The ability to see some guns might help with knowing you need to stay away from the ship or die

**Comment by Zitchas** [comment]
> I agree w QuantumShark, also a weapon against a group of heavy warship doesnt seem like it'd be useful except against the player, and even then it'd be a joke compared to things like nukes

The player starts out solo, and often goes the game solo. The AI deathballs in virtually every battle, and the heavies are particularly bad for it. So a weapon for dealing with groups of heavies predominantly favors solo players. Agreed, however, that it does penalize fleet players just as much as it does fleet NPCs.

Either way, I recognize that I am in the minority in wanting better tools for small ships to take on heavies with, and don't see the point in making everyone better capable of taking out small ships. 

Likewise, I'm of the opinion that a shuttle should take a heavy rocket about as well as the average sedan takes a direct hit by an RPG. Which is to say, dead. Blown utterly to smithereens. It's a civilian light transport being hit by top end military munitions. It has no business being able to tank a direct hit in the least.

**Comment by Hurleveur** [comment]
> Likewise, I'm of the opinion that a shuttle should take a heavy rocket about as well as the average sedan takes a direct hit by an RPG. Which is to say, dead. Blown utterly to smithereens. It's a civilian light transport being hit by top end military munitions. It has no business being able to tank a direct hit in the least.

I dont think thats good gameplay for new players

**Comment by Quantumshark** [comment]
> Either way, I recognize that I am in the minority in wanting better tools for small ships to take on heavies with, and don't see the point in making everyone better capable of taking out small ships.

The "better tool for small ships to take on heavies with", if you're willing to use ammo-limited weapons, is the torpedo pod. It's much better in that role than heavy rockets are, with or without these changes. The heavy rocket does not need to be more of the same. If there are specialised anti-HW weapons, there should also be specialised anti-fighter weapons.

**Comment by Zitchas** [comment]
> If there are specialised anti-HW weapons, there should also be specialised anti-fighter weapons.

No, specialized weapons should only exist where there is a need for them. Currently, regular weapons are so overwhelmingly effective that the general complaint is that these entire categories of ships are useless and pointless (fighters, drones, light ships in general). This is not an indicator that we need a specialized weapon to take them out faster, or that it needs optimization to improve its performance in this role.

The balance of the game is already overwhelmingly in favor of larger ships, it is explicitly bad for gameplay to have and/or optimize weapons to make them better at taking out categories of ships that have already been deemed useless.

**Comment by Hecter94** [comment]
https://github.com/endless-sky/endless-sky/blob/cc42ea87c690aee9ab22cc050963a49d02f63953/data/human/intro%20missions.txt#L405

Should we also change this line? Just say two heavy rocket hits will nearly disable?

**Review by Hecter94 — APPROVED** [review]
This seems like a good change to me. 
We've gotten a lot of complaints over the years about instant deaths, and heavy rockets were one of the biggest contributors to that. 
This also makes heavy rockets more in-line with what they are purported to be, and expands the roster of AoE weapons which we have far too few of. 

Overall, seems like a lot of good changes.

**Comment by AvianGeneticist** [comment]
Blast Radii in general have been far too small in my opinion (in many cases they more often serve to deal recoil damage rather than actually hitting multiple targets due to how spaced out ships are in Endless Sky), so I'm excited to see how this balance change might shape other AoE weapons in the future.
